rpi-eeprom-config

```diff
@@ -22,18 +27,128 @@ MAGIC_MASK = 0xfffff00f
 FILE_MAGIC = 0x55aaf11f # id for modifiable file, currently only bootconf.txt
 FILE_HDR_LEN = 20
 FILENAME_LEN = 12
+TEMP_DIR = None
+
+def exit_handler():
+    """
+    Delete any temporary files.
+    """
+    if TEMP_DIR is not None and os.path.exists(TEMP_DIR):
+        tmp_image = '%s/pieeprom.upd' % TEMP_DIR
```

Comment:
Doesn't really make any difference, but personally I prefer doing things like this with `os.path.join(TEMP_DIR, 'pieeprom.upd')` rather than using string-formatting.

Reply:
I'll think about it, os.path.join seems rather wordy

Reply:
I decided to make the change in-case someone manages to sneak in a wierd path / extra separators